dev/contr.md

```diff
@@ -0,0 +1,15 @@
+# Contributing
+
+All contributions must be made via a Pull Request to the main GitHub-Repo.
+
+For a pull request to get merged, the following conditions must be fulfilled:
+
+- all applicable pipelines must succeed
+- at least two approvals must be made for the patch to be merged after 24 hours, or at least four approvals must be made to merge the patch immediately
+- the PR must neither be controversial or blocked (indicated by tags)
```

Comment:
I would also add a notice, that changes in the, database scheme and especially in areas that were disputed before, also require 4 approvals. What do you think?

Reply:
I think that should be covered by the controversial tag

Reply:
Maybe we should add a "fundamental"-state, requiring a deeper discussion and 4 approvals?